python/src/KarhunenLoeveSVDAlgorithm_doc.i.in

```diff
@@ -24,24 +24,39 @@ centeredSample : bool, default=False
     
 Notes
 -----
-The Karhunen-Loeve SVD algorithm solves the Fredholm problem  associated to the covariance function :math:`C`: see :class:`~openturns.KarhunenLoeveAlgorithm` to get the notations.
-
-The SVD approach is a particular case of the quadrature approach (see :class:`~openturns.KarhunenLoeveQuadratureAlgorithm`)  where we consider the functional basis  :math:`((\vect{\theta}_p^j(\vect{s}))_{1 \leq j \leq d, 1 \leq p \leq P}` of :math:`L^2(\cD, \Rset^d)` defined on :math:`\cD` by:
+The Karhunen-Loeve SVD algorithm solves the Fredholm problem  associated to the
+covariance function :math:`C`: see :class:`~openturns.KarhunenLoeveAlgorithm`
+to get the notations.
+If the mesh is regular, then this decomposition is equivalent to the principal
+component analysis (PCA) (see [pearson1907]_, [hotelling1933]_, [jackson1991]_, [jolliffe2002]_).
+In other fields such as computational fluid dynamics for example,
+this is called proper orthogonal decomposition (POD) (see [luo2018]_).
+
+The SVD approach is a particular case of the quadrature approach (see
+:class:`~openturns.KarhunenLoeveQuadratureAlgorithm`)  where we consider the
+functional basis  :math:`((\vect{\theta}_p^j(\vect{s}))_{1 \leq j \leq d, 1 \leq p \leq P}` of
+:math:`L^2(\cD, \Rset^d)` defined on :math:`\cD` by:
 
 .. math:: 
 
     \vect{\theta}_p^j(\vect{s})= \left[\mat{C}(\vect{s}, \vect{s}_p) \right]_{:, j}
```

Comment:
I don't believe $\boldsymbol{s}_p$ has been defined anywhere.

Reply:
I agree. I do not understand this equation. Notice that neither $d$ nor $P$ are defined. I think that someone who understands this may be of some help here. @adutfoy : any clue?

Reply:
This is defined in KarhunenLoeveSVDAlgorithm. A new link was inserted.